src/CSET/operators/write.py

```diff
@@ -24,19 +25,24 @@
 
 
 def write_cube_to_nc(
-    cube: Union[iris.cube.Cube, iris.cube.CubeList], filename: Path = None, **kwargs
+    cube: Union[iris.cube.Cube, iris.cube.CubeList],
+    filename: str = None,
+    overwrite: bool = False,
+    **kwargs,
 ) -> str:
     """Write a cube to a NetCDF file.
 
-    This operator expects an iris cube object that will then be passed to MET
-    for further processing.
+    This operator expects an iris cube object that will then be saved to disk.
 
     Arguments
     ---------
     cube: iris.cube.Cube | iris.cube.CubeList
         Data to save.
-    filename: Path, optional
+    filename: str, optional
```

Comment:
The arguments to these operators come in from the recipe files, and YAML doesn't have a specific path reprisentation as such it is being passed in as a string.

Also, this specific argument will likely mainly be used with a single filename glob pattern, instead of a path of multiple components.